app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -321,7 +321,7 @@ class MainActivity : ComponentActivity() {
         if (result.resultCode == RESULT_OK) {
             firebaseUser = FirebaseAuth.getInstance().currentUser
             firebaseUser?.let {
-                val user = com.ucmobiledevelopment.freeways.dto.User(it.uid, it.displayName)
+                val user = User(it.uid, it.displayName)
```

Comment:
Path was redundant as it was already imported.

Reply:
Ah, yes.  Good one.